libcst/codemod/visitors/_remove_imports.py

```diff
@@ -374,12 +374,19 @@ def _process_importfrom_aliases(
                 if len(names_to_keep) != 0:
                     # there is a previous import alias
                     prev = names_to_keep[-1]
-                    names_to_keep[-1] = prev.with_deep_changes(
-                        whitespace_after=_merge_whitespace_after(
-                            prev.comma.whitespace_after,
-                            comma.whitespace_after,
+                    if isinstance(prev.comma, cst.Comma):
+                        prev = prev.with_deep_changes(
+                            whitespace_after=_merge_whitespace_after(
+                                prev.comma.whitespace_after,
+                                comma.whitespace_after,
+                            )
                         )
```

Comment:
This is not tested. Can you add a test case?

https://codecov.io/gh/Instagram/LibCST/src/9d76eadaa3b5a52cadeb75960b587b446a0f508f/libcst/codemod/visitors/_remove_imports.py